tests/dspa_v2_test.go

```diff
@@ -91,3 +91,81 @@ func (suite *IntegrationTestSuite) TestDSPADeployment() {
 		}
 	})
 }
+
+func (suite *IntegrationTestSuite) TestDSPADeploymentWithK8sNativeApi() {
```

Comment:
I think this test is not asserting that the webhook is deployed because `suite.DSPA.Spec.APIServer.PipelineStorage = "true"` is not correct.